app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt

```diff
@@ -297,6 +303,16 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
                 navUrlInput.setText(it.urlBarText)
             }
         })
+
+        toolbarViewModel.events.observe(lifeCycleOwner, Observer {
```

Comment:
Also events related to pinning from the tile itself, not just the pin button. I'm thinking that we might want this to eventually belong to the overlay, but need to think more about the hints/notifications that Amin designed into the 4.0 overlay (per channel).

Reply:
Added a note to move these when possible.